graph-sync/base/kustomization.yaml

```diff
@@ -1,11 +1,12 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+commonLabels:
+  app.kubernetes.io/name: thoth
+  app.kubernetes.io/component: graph-sync
+  app.kubernetes.io/managed-by: kustomize-ksops
```

Comment:
this needs to be some different value ;)

Reply:
Change the managed-by to aicoe-thoth-devops.